src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+  let usedKeys = '';
+
+  for (const key in robot) {
+    reversedRobot[robot[key]] = key;
+
+    if (usedKeys.includes('' + robot[key])) {
+      return null;
+    }
```

Comment:
better use `Object.hasOwnProperty` - it makes only one operation (checking the key), but `includes` is looping through the whole string